### ISSUE TITLE
Padding bugs on data preprocess

### ISSUE THREAD

**Issue by AppleHolic** [dhlee347/pytorchic-bert#10]
On this code [line](https://github.com/dhlee347/pytorchic-bert/blob/a2e0b8aace370af0f951814f7aad525a9dca8722/pretrain.py#L152),
the pad index 0 is same with first segment index.
So, it may not offer segment information exactly.

**Comment by dhlee347** [comment]
I merged your pull requests.

**Comment by yeachan-kr** [comment]
The pad index 0 does not affect to the accuracy since the paddings will be masked. 
Further, such requests cause the out-of-bound problem when loading the pre-trained models.
